### ISSUE TITLE
Action List Item: On accessing 1st ‘Action’ button as highlighted, information about the whole list items is made at once.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3832]
Description: On accessing 1st ‘Action’ button as highlighted, information about the whole list items is made at once.

Expected: Information should be clear & precisely conveyed by the screen reader, on accessing the UI element.

Screenshot:

![image](https://user-images.githubusercontent.com/32538291/98774577-496f3180-2411-11eb-8dfb-2af8752a43c1.png)

**Comment by sKudum** [comment]
Update the message information
